cmds/fusermount/fusermount_linux.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
```

Comment:
Having a separate `_linux.go` file doesn't mean much when the other file already has `// +build linux`.